### PR TITLE
DHE documentation update sync.

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit SvenDowideit@home.org.au

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@fredlf - this is for you :)

File: docs/sources/docker-hub-enterprise/configuration.md [hunk 1]
```diff
@@ -267,42 +269,85 @@ by the [Registry 2.0](http://docs.docker.com/registry/configuration/).
...
 ## Authentication
 
+The "Authentication" settings tab lets DHE administrators control access
+to the DHE web admin tool and to the DHE Registry.
+
 The current authentication methods are `None`, `Basic` and `LDAP`.
 
-The `Basic` setting includes:
+> **Note**: if you have issues logging into the DHE admin web interface after changing the authentication
+> settings, you may need to use the [emergency access to the DHE admin web interface](./adminguide.md#Emergency-access-to-the-dhe-admin-web-interface).
```
Comment (thaJeztah) [inline docs/sources/docker-hub-enterprise/configuration.md]: Just checking; does the anchor (`#Emergency-access..`) actually start with a capital?

Comment (fredlf) [inline docs/sources/docker-hub-enterprise/configuration.md]: It does. It's a section heading. Thx.

**Comment by fredlf** [comment]
LGTM

**Pull request merged**